src/main.ts

```diff
@@ -82,13 +84,39 @@ function fileWatch(configs: webpack.Configuration[], args: any): Promise<void> {
 			}
 			if (stats) {
 				const runningMessage = args.serve ? `Listening on port ${args.port}` : 'watching...';
-				logger(stats.toJson(), configs, runningMessage);
+				logger(stats.toJson(), configs, args.target === 'lib', runningMessage);
 			}
 			resolve();
 		});
 	});
 }
 
+let libEntryPath: string;
+function getLibEntryPath(elements: Array<{ name: string; path: string }>) {
+	if (libEntryPath) {
+		return libEntryPath;
+	}
+
+	const basePath = process.cwd();
+	libEntryPath = path.join(basePath, 'src', 'main.ts');
+	if (!fs.existsSync(libEntryPath)) {
```

Comment:
As per the comment above, if we use the `element.path` as the module to load we shouldn't need to create any temporary files.